server/plugins/Shell.js

```diff
@@ -49,6 +49,8 @@ function run(trigger, scope, data, config, callback) {
     }else{
       logger.main.info('File '+full_file_name+' not exist');
     }
+
+    callback(data);
   }
```

Comment:
Missing
```
} else {
  callback();
}
```
for the main if in function, because plugin must always call `callback`
See [here](https://github.com/pagermon/pagermon/tree/v0.3.9/server/plugins).